mdm/api/src/main/java/com/gentics/mesh/core/data/dao/RootDao.java

```diff
@@ -297,16 +297,6 @@ default long computeCount(R root) {
 	 */
 	void delete(R root, L element, BulkActionContext bac);
 
-	/**
-	 * Update the element using the action context information.
-	 *
-	 * @param ac
-	 * @param batch
-	 *            Batch to which entries will be added in order to update the search index.
-	 * @return true if the element was updated. Otherwise false
-	 */
-	boolean update(R root, L element, InternalActionContext ac, EventQueueBatch batch);
```

Comment:
This was removed since I couldn't find any usages.

Reply:
I get it back. There are entities, that depend on the root to be resolved, and so this method may potentially get called.